docs/source/index.rst

```diff
@@ -0,0 +1,235 @@
+        "RETURN n"
+    )
+    parameters = {
+        "id": 0,
+        "vector": vector,
+    }
+    driver.execute_query(insert_query, parameters)
+
+
+.. note::
+
+    Assumption: Neo4j running with a defined vector index
+
+Development
+===========
+
+********************
+Install dependencies
+********************
+
+.. code:: bash
+
+    poetry install
+
+***************
+Getting started
+***************
+
+~~~~~~
+Issues
+~~~~~~
+
+If you have a bug to report or feature to request, first
+`search to see if an issue already exists <https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments>`_.
+If a related issue doesn't exist, please raise a new issue using the relevant
+`issue form <https://github.com/neo4j/neo4j-genai-python/issues/new/choose>`_.
+
+If you're a Neo4j Enterprise customer, you can also reach out to `Customer Support <http://support.neo4j.com/>`_.
+
+If you don't have a bug to report or feature request, but you need a hand with
+the library; community support is available via `Neo4j Online Community <https://community.neo4j.com/>`_
+and/or `Discord <https://discord.gg/neo4j>`_.
+
+~~~~~~~~~~~~
+Make changes
+~~~~~~~~~~~~
+
+1. Fork the repository.
+2. Install Python and Poetry.
+3. Create a working branch from `main` and start with your changes!
+
+~~~~~~~~~~~~
+Pull request
+~~~~~~~~~~~~
+
+When you're finished with your changes, create a pull request, also known as a PR.
+
+-   Ensure that you have `signed the CLA <https://neo4j.com/developer/contributing-code/#sign-cla>`_.
+-   Ensure that the base of your PR is set to `main`.
+-   Don't forget to `link your PR to an issue <https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue>`_
+    if you are solving one.
+-   Enable the checkbox to `allow maintainer edits <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork>`_
+    so that maintainers can make any necessary tweaks and update your branch for merge.
+-   Reviewers may ask for changes to be made before a PR can be merged, either using
+    `suggested changes <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>`_
+    or normal pull request comments. You can apply suggested changes directly through
+    the UI, and any other changes can be made in your fork and committed to the PR branch.
+-   As you update your PR and apply changes, mark each conversation as `resolved <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations>`_.
+
+*********
+Run tests
+*********
+
+Open a new virtual environment and then run the tests.
+
+.. code:: bash
+
+    poetry shell
+    pytest
+
+
```

Comment:
I have this section updated in my external vector store PR, so make wait for that.